### PR TITLE
Fixes #18482: Move health check API in /system/

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/18482

### CODE REVIEW

File: webapp/sources/api-doc/openapi.src.yml [hunk 1]
```diff
@@ -153,6 +151,8 @@ paths:
     $ref: paths/system/archives-restore.yml
   "/system/archives/{archiveKind}/zip/{commitId}":
     $ref: paths/system/archives-zip.yml
+  "/system/healthcheck":
+    $ref: paths/healthcheck/all.yml
```
Comment (amousset) [inline webapp/sources/api-doc/openapi.src.yml:155]: the yml file should be moved too

**Comment by ElaadF** [comment]
Commit modified

File: webapp/sources/api-doc/openapi.src.yml [hunk 2]
```diff
@@ -153,6 +151,8 @@ paths:
     $ref: paths/system/archives-restore.yml
   "/system/archives/{archiveKind}/zip/{commitId}":
     $ref: paths/system/archives-zip.yml
+  "/system/healthcheck":
+    $ref: paths/system/healthcheck.yml
```
Comment (amousset) [inline webapp/sources/api-doc/openapi.src.yml:155]: You also need to remove the "Health check" tag from the openapi.src.yml and healthcheck.yml files

**Comment by ElaadF** [comment]
Commit modified

**Review by fanf — APPROVED** [review]

**Pull request merged**